ldd-check/pipelines/test/tw/ldd-check.yaml

```diff
@@ -0,0 +1,30 @@
+name: ldd-check
+
+needs:
+  packages:
+    - ldd-check
+
+inputs:
+  files:
+    description: |
+      The files to run `ldd` on and check for missing deps.
+    required: false
+  packages:
+    description: |
+      Check all binaries in these installed packages.
+      Use "none" to disable explicit package check.
```

Comment:
implicit?

Reply:
changed. i hope reasonable now.
thanks for reading.